src/v2/services/IWalletService.ts

```diff
@@ -19,4 +22,27 @@ export interface IWalletService {
     successMessage?: string,
     transactionTip?: number
   ): Promise<string | null>;
+
+  signAndSendWithCustomSignature({
```

Comment:
[signAndSend](https://github.com/AstarNetwork/astar-apps/blob/2f553afa191c401a50cac9dd0bad9a4db5c92be2/src/v2/services/implementations/PolkadotWalletService.ts#L36) method from PolkadotWalletService should work with the lockdrop account. You can take a look at [this](https://github.com/AstarNetwork/astar-apps/pull/653) branch (sending tokens from lockdrop account).

Reply:
CC @bobo-k2

Reply:
@sirius651 

> [signAndSend](https://github.com/AstarNetwork/astar-apps/blob/2f553afa191c401a50cac9dd0bad9a4db5c92be2/src/v2/services/implementations/PolkadotWalletService.ts#L36) method from PolkadotWalletService

Ah sorry, it should be signAndSend method from [MetamaskWalletService](https://github.com/AstarNetwork/astar-apps/blob/2f553afa191c401a50cac9dd0bad9a4db5c92be2/src/v2/services/implementations/MetamaskWalletService.ts#L35)

Anyway, I think you can use it like [this](https://github.com/AstarNetwork/astar-apps/blob/feat/light-client/src/v2/services/implementations/TokenTransferService.ts#L26)

Reply:
does that mean we can remove useCustomSignature hook further?

Reply:
Yes, I think we can remove that hook eventually